WordPress/Classes/Utility/ContextManager.m

```diff
@@ -33,6 +34,7 @@ - (instancetype)init
 {
     self = [super init];
     if (self) {
+        _derivedContextMap = [NSMapTable strongToStrongObjectsMapTable];
```

Comment:
Maybe use the setter vs assigning to the ivar directly, just as a best practice?

Reply:
Good call, thank you Eric!